CONTRIBUTING.md

```diff
@@ -0,0 +1,255 @@
+
+The folder should have a `README.md` file with descriptive information for others to start using your code or tutorial notebooks.
+
+Finally, the MONAI tutorial has a [README file](README.md) to communicate the important information and provide an overview of the tutorials and examples. For new tutorials, please add a new entry to the [List](README.md#4-list-of-notebooks-and-examples) of notebooks and examples](README.md#4-list-of-notebooks-and-examples).
+
+### Add license
+
+All source code and notebook files should include copyright information at the top of the file.
+
+NOTE: for Jupyter Notebook `.ipynb` files, the copyright information should appear at the top of the first markdown cell.
+There are extra two spaces at the end of each line to ensure no line auto-wrap in the markdown rendering in the display.
+
+```markdown
+Copyright (c) MONAI Consortium
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+```
+
+### Create a notebook
+
+Jupyter Notebook is the preferred way of writing a new MONAI tutorial because we encourage contributors to visualize the outputs and keep the records with the code.
+
+Writing a notebook is easy and flexible, but we require all tutorial notebooks to start with the following three sections:
+
+- Title of the notebook with [licensing information](#add-license) in one markdown cell
+- A markdown cell with `## Setup environment` and a code cell that executes `pip install` shell commands with the exclamation mark`!`` to install all the packages necessary for your tutorial.
+
+    For example, we install [MONAI with extra dependencies](https://docs.monai.io/en/stable/installation.html#installing-the-recommended-dependencies) and `matplotlib` to set up the environment for the [mednist_tutorial](./2d_classification/mednist_tutorial.ipynb):
+
+    ```
+    !python -c "import monai" || pip install -q "monai-weekly[pillow, tqdm]"
+    !python -c "import matplotlib" || pip install -q matplotlib
+    ```
+
+- A markdown cell with `## Setup imports` and a code cell that contains **ALL** import statements and ends with `monai.config.print_config()`:
+
+    ```python
+    import numpy as np
+
+    from monai.config import print_config
+    from monai.data import DataLoader
+    ...
+
+    print_config()
+    ```
+
+The tutorial provides [useful templates](.github/contributing_templates/notebook/README.md) for contributors to start with.
```

Comment:
Maybe change "The tutorial" to "The guideline"?

Thanks.

Reply:
I changed the sentence to:
“Following this guideline, we prepare some [templates](.github/contributing_templates/notebook/README.md) for contributors to start with.”

Thanks!